invenio_accounts/translations/cs/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,171 @@
+# Translations template for invenio-accounts.
+# Copyright (C) 2016 CERN
+# This file is distributed under the same license as the invenio-accounts
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: invenio-accounts 1.0.0a14.dev20160704\n"
+"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
+"POT-Creation-Date: 2016-08-18 10:27+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Jiří Kunčar <jiri.kuncar@gmail.com>, 2016\n"
+"Language-Team: Czech (https://www.transifex.com/inveniosoftware/teams/23537/cs/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+"Language: cs\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+
+#: invenio_accounts/admin.py:68
+msgid "Current Login IP"
+msgstr "Současná IP připojení"
+
+#: invenio_accounts/admin.py:69
+msgid "Last Login IP"
+msgstr "IP posledního připojení"
+
+#: invenio_accounts/admin.py:72
+msgid "Inactivate"
+msgstr "Inaktivovat"
```

Comment:
@helix84 you should be a part of translation team on transifex where you can review it (https://www.transifex.com/inveniosoftware/teams/23537/cs/). Thanks!